ansible_collections/arista/avd/examples/campus-fabric/group_vars/NETWORK_SERVICES.yml

```diff
@@ -1,107 +1,107 @@
 ---
-### group_vars/DC1_NETWORK_SERVICES.yml
+### group_vars/NETWORK_SERVICES.yml
 
 tenants:
-  - name: MY_FABRIC
+  - name: ACME_SITE1
     vrfs:
       - name: default
         svis:
           - id: 110
-            name: 'IDF1-Data'
-            tags: ["110"]
+            name: IDF1-Data
+            tags: [110]
```

Comment:
tag is a string in essence - keep it quoted

Reply:
For tags, we can either leverage ints or strings as tag names. They will be converted to strings either way. It seems like unnecessary quotes